website_sale_vat_required/website_order_view.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
 <data>
-    <template id="saas_users" inherit_id="website_sale.checkout">
-        <label class="control-label" for="vat" position="attributes">
+    <template id="vat_checkout" inherit_id="website_sale.checkout">
+        <xpath expr="//div[@class='container oe_website_sale']/form/div[@class='row']/div[@class='col-md-8 oe_cart']/div[@class='row']/div[4]/label" position="attributes">
```

Comment:
@pedrobaeza I didn't find a way to simplify it. Do you have suggestions?

@nhomar do you have an example about ```option="True"```?